sources/web/datalab/polymer/components/datalab-sidebar/datalab-sidebar.ts

```diff
@@ -24,17 +24,27 @@ class SidebarElement extends Polymer.Element {
    */
   public page: string;
 
+  private _showDataTab: boolean;
```

Comment:
You could also remove the link from the sidebar, then manually typing `/data` in the address bar would open the data page, even without this code here.